src/components/InputTextList.tsx

```diff
@@ -80,7 +82,7 @@ export default function InputTextList(props) {
                         })
                     }
                 >
-                    <PlusOutlined /> Add another input
+                    <PlusOutlined /> {t("claimReviewForm:addInput")}
```

Comment:
I think it would be useful to move the i18n string to the prop and create a prop `addButtonLabel` or something like that